src/database.ts

```diff
@@ -5,13 +5,21 @@ import { fileURLToPath } from 'node:url';
 import './env.ts';
 
 let settings: Knex.Config | null = null;
-const db: Knex = knex(getSettings());
+const db: Knex = knex(await getSettings());
 
 export async function init() {
-
   console.info('Running database migrations');
-  await db.migrate.latest();
-
+  await db.migrate.latest()
+    .then(() => {
+      if (process.env.NODE_ENV === 'development') {
+        console.info('Running database seeds');
+        return db.seed.run();
+      }
+    })
+    .catch((error) => {
+      console.error('Migrations failed', error);
+      process.exit(1);
+    });
 }
```

Comment:
This should all be reverted.

Reply:
Ah I thought the server script command was necessary in order for this to work.

But accordingly if I'm changing the seed config in settings, it should run when the `knex seed:run ` is called I hope

Reply:
Yes it should. `knex seed:run` should run all the seed files unconditionally. I think seed files are very useful to get a whole bunch of test data, but there really should be no need for any configuration. This PR just needs to be simplified a bit